packages/sit-onyx/src/composables/useAnchorPositionPolyfill.ts

```diff
@@ -38,6 +31,24 @@ type UseAnchorPositionPolyfillOptions = {
   offset?: number;
 };
 
+/**
+ * SSR safe composable for checking whether the browser supports the Anchor API.
+ * TODO: can be removed after anchor is implemented in all common browsers.
+ */
+export const useSupportsAnchorApi = () => {
+  const USERAGENT_SUPPORTS_ANCHOR_API = ref(true);
+
+  onBeforeMount(() => {
+    USERAGENT_SUPPORTS_ANCHOR_API.value =
+      "CSS" in globalThis &&
+      typeof CSS !== "undefined" &&
+      CSS.supports("anchor-name: --test") &&
+      CSS.supports("position-area: top");
+  });
+
+  return { USERAGENT_SUPPORTS_ANCHOR_API };
```

Comment:
Only (global) constants should be capitalized

```suggestion
  return { useragentSupportAnchorApi };
```